modern-gpus/intel-gpu.md

```diff
@@ -11,12 +11,10 @@ More info:
 ## Native Intel iGPUs
 
 ### **Ivy Bridge 3XXX**
-#### Highest Supported OS: Catalina(10.15)
-#### Initial Supported OS: Lion(10.7)
+#### Highest Supported OS: Big Sur (11)
+#### Initial Supported OS: Lion (10.7)
 
-Regarding the HD 4000, it's completely native with Catalina(Though no longer supported with macOS 11: Big Sur). The HD 2500 on the other hand only has partial support in Mojave for quick sync features as hardware acceleration is [unsupported](https://olarila.com/forum/viewtopic.php?t=7714) so you will need a compatible dGPU for display purposes. Initial supported introduced with macOS 10.8
-
-* Note: As of macOS 11.3, the HD 4000 drivers are still present in Big Sur. Please keep in mind future OS updates and releases may remove these drivers.
+Regarding the HD 4000, it's completely native though no longer supported with macOS Monterey. Note that you must be using an Ivy Bridge-compatible SMBIOS. The HD 2500, on the other hand, only has partial support in Mojave for quick sync features, but hardware acceleration is unsupported, so you will need a compatible dGPU for display purposes.
```

Comment:
```suggestion
The HD4000 is supported natively up to macOS Big Sur, with support dropped in macOS Monterey. Note that you must be using an Ivy Bridge-compatible SMBIOS. The HD 2500, on the other hand, only has partial support in Mojave for quick sync features. As the HD 2500 does not support hardware acceleration, you will need a compatible dGPU for display purposes.
```

"Ivy Bridge-compatible SMBIOS" is a bit of a misnomer. We have SMBIOS recommendations in the install guide for both Ivy bridge desktops and laptops which will work. They're Haswell though.

Reply:
Resolved with bd390f8 and 3472111